src/test_l1_tvSettings.c

```diff
@@ -13967,7 +14248,7 @@ void test_l1_tvSettings_negative_SaveCMS (void)
 	if (extendedEnumsSupported == true)
 	{
 		/* Step 21: Calling tvsettings SaveCMS and expecting the API to return tvERROR_INVALID_STATE */
-		result = SaveCMS((tvVideoSrcType_t)Configfile.videoSrcStruct.videoSourceValue[0], Configfile.picmodeStruct.pqValue[0], (tvVideoFormatType_t)Configfile.videoFormtStruct.videoFormatValue[0], (tvComponentType_t)Configfile.componentColor.modeId[0], (tvDataComponentColor_t)Configfile.componentColor.modeId[0], 1);
+		result = SaveCMS((tvVideoSrcType_t)videoSource, pqValue, (tvVideoFormatType_t)videoFormat, (tvComponentType_t)Configfile.componentColor.modeId[0], (tvDataComponentColor_t)Configfile.componentColor.modeId[0], 1);
```

Comment:
Configfile.componentColor.modeId is still present

Reply:
Replaced